Realm/Realm/Configurations/SyncConfiguration.cs

```diff
@@ -187,7 +191,7 @@ internal override async Task<Realm> CreateRealmAsync(RealmSchema schema, Cancell
             }
         }
 
-        internal Native.SyncConfiguration ToNative()
+        internal Native.SyncConfiguration CreateSyncConfiguration()
```

Comment:
I liked the previous name better. The new name is confusing because this method is already exposed on a `SyncConfiguration` class. I guess you're modeling it after `CreateConfiguration` but that is also terrible - must have been Yavor who introduced it 😝 If you want to keep `CreateConfiguration`/`CreateSyncConfiguration`, at least throw in `Native` in there.